### DISCUSSION TITLE
怎么加载自己本地的高程服务和影像服务

### DISCUSSION THREAD

**Discussion by styleWql** [zouyaoji/vue-cesium#192] [Q&A] [answered]
怎么加载自己本地的高程服务和影像服务

**Comment by zouyaoji** [comment, marked as answer]
用 `vc-imagery-provider-urltemplate` 组件基本都能加。
xyz瓦片用 `vc-imagery-provider-tms`组件。

地形用 `vc-terrain-provider-cesium` 组件，通过url属性传进去，写到json所在文件夹就可以了。

**Reply by styleWql** [reply]
这些组件在文档里面有详细的讲解吗，我怎么没有找到呢

**Reply by zouyaoji** [reply]
没有~~

**Reply by styleWql** [reply]
官方文档有点难嚼
